### ISSUE TITLE
Healing Nanites

### ISSUE THREAD

**Issue by Growlkore** [Better-Scenes/RimConnect-mod#27]
Healing nanites cures brain scars.

**Comment by InfinitySamurai** [comment]
Is this a problem? Shouldn't be hard to fix if it is

**Comment by InfinitySamurai** [comment]
Looks like this is caused by the crush damage removal. Crush damage has a range of effects one of which is "mangled scar" which can cause the brain damage. 

```
    <comps>
      <li Class="HediffCompProperties_TendDuration">
        <labelTendedWell>bandaged</labelTendedWell>
        <labelTendedWellInner>sutured</labelTendedWellInner>
        <labelSolidTendedWell>set</labelSolidTendedWell>
      </li>
      <li Class="HediffCompProperties_Infecter">
        <infectionChance>0.15</infectionChance>
      </li>
      <li Class="HediffCompProperties_GetsPermanent">
        <permanentLabel>mangled scar</permanentLabel>
      </li>
    </comps>
```

**Comment by InfinitySamurai** [comment]
I think the easiest solution is to remove this from the list of injuries that can be healed even though colonists can have a mild form of crush damage. I don't want to deal with the extra complexity of handling "less severe" versions of hediffs

**Comment by InfinitySamurai** [comment]
Fixed in f779cf1
